RecoTauTag/RecoTau/python/hpsPFTauDiscriminationByAMuonRejectionSimple_cff.py

```diff
@@ -3,12 +3,12 @@
 from RecoTauTag.Configuration.HPSPFTaus_cff import hpsPFTauDiscriminationByMuonRejection3
 
 hpsPFTauDiscriminationByLooseMuonRejectionSimple = hpsPFTauDiscriminationByMuonRejection3.clone(
-    PFTauProducer=cms.InputTag("hpsPFTauProducer"),
-    HoPMin=cms.double(0.1), #use smaller value than with AOD as raw energy is used
-    doCaloMuonVeto=cms.bool(False), #do not use it until tuned
-    srcPatMuons=cms.InputTag("slimmedMuons"),
-    maxNumberOfSTAMuons=cms.int32(-1),
-    maxNumberOfRPCMuons=cms.int32(-1),
+    PFTauProducer = cms.InputTag("hpsPFTauProducer"),
+    HoPMin = hpsPFTauDiscriminationByMuonRejection3.IDWPdefinitions[0].HoPMin,
+    doCaloMuonVeto = cms.bool(True),
+    srcPatMuons = cms.InputTag("slimmedMuons"),
+    maxNumberOfSTAMuons = cms.int32(-1),
+    maxNumberOfRPCMuons = cms.int32(-1),
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
```suggestion
    PFTauProducer = "hpsPFTauProducer",
    HoPMin = hpsPFTauDiscriminationByMuonRejection3.IDWPdefinitions[0].HoPMin,
    doCaloMuonVeto = True,
    srcPatMuons = "slimmedMuons",
    maxNumberOfSTAMuons = -1,
    maxNumberOfRPCMuons = -1,
```